cmd/shell.go

```diff
@@ -30,6 +30,8 @@ import (
 	blog "github.com/letsencrypt/boulder/log"
 )
 
+var command = path.Base(os.Args[0])
```

Comment:
I think it would be slight better to extract this to a function rather than a variable. That way we know no-one can change its value, either accidentally or on purpose.